### DISCUSSION TITLE
Cannot get gw to install on M1/Monterey OS X

### DISCUSSION THREAD

**Discussion by icscolor** [keirf/greaseweazle#184] [Q&A]
So far I haven't been able to get gw to install. I've installed python3 but I can't get past this:

/Library/Frameworks/Python.framework/Versions/3.3/bin/python3: No module named pip

Every time I try to install pip it is failing.

Has anyone else run into this?

**Comment by keirf** [comment]
System Python on Mac is a mess. Perhaps investigate pyenv / virtualenv and run greaseweazle in a clean local python env of its own? You may be able to install this via Homebrew.

**Reply by sideburn** [reply]
Yeah I tried pyenv too and had trouble. I'll try windows through parallels.

**Reply by keirf** [reply]
Unfortunately I'm weaned off Apple some years now and can't run Monterey. But there's an xkcd cartoon which illustrates the mess that is Python on mac.

**Reply by d235j** [reply]
Apple stopped providing Python with the latest macOS version (12.3) because it was such a mess. You’ll need to install it using MacPorts, Homebrew, or the official package.

**Reply by d235j** [reply]
The above path (version 3.3, in /Library/Frameworks) is likely a previously manually installed one. I would recommend deleting that directory and starting over. At this point 3.3 is too old to do most anything and is also unsupported on M1.

**Reply by icscolor** [reply]
Well I tried my windows 10 box (parallels on mac didn't work either) and now gw.exe/info works but thats it. whenever i try to access the disk with "info" or "erase" it says getFluxStatus: no index

I tried both the ribbon cable in the amiga and the non twisted cable i got with the greaseweazle...
I am also externally powering the floppy with the amiga PSU.

**Reply by keirf** [reply]
Is it a PC drive? Try `gw erase --drive b`

**Reply by icscolor** [reply]
no its an amiga 2000 drive. I bought the GW from amigakit along with the power and straight through drive cable. neither are working so i am powering the drive form the PSU.

**Reply by icscolor** [reply]
<img width="544" alt="Screen Shot 2022-04-03 at 1 30 23 PM" src="https://user-images.githubusercontent.com/82837542/161447399-ba5a0e1d-925b-42d4-bab9-b3f374136b2c.png">
<img width="590" alt="Screen Shot 2022-04-03 at 1 32 02 PM" src="https://user-images.githubusercontent.com/82837542/161447404-ce350054-77ab-456d-80e2-39217a109805.png">
<img width="993" alt="Screen Shot 2022-04-03 at 1 32 41 PM" src="https://user-images.githubusercontent.com/82837542/161447406-559f2036-7d2e-4fdc-8bd3-992320e927e3.png">

**Reply by keirf** [reply]
Do you happen to have a PC drive? They are easier to power and generally produce better dumps than old Amiga drives, too.

**Reply by keirf** [reply]
Actually you probably just need `gw erase --drive 0`

**Reply by icscolor** [reply]
Bingo! gw erase --drive 0 worked. OK so now what's the key to putting a .adf disk image onto the floppy?

**Reply by keirf** [reply]
`gw write --drive 0 my.adf` or whatever your adf is called.
